.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve Ignite
+
```

Comment:
Can we also include a label here?

Reply:
Well, I reported bug may not be ignite bug :)